src/os/bluestore/bluefs_types.h

```diff
@@ -308,6 +308,7 @@ struct bluefs_super_t {
   std::optional<bluefs_layout_t> memorized_layout;
 
   std::vector<uint64_t> bluefs_max_alloc_size;
+  bool wal_v2;
```

Comment:
Let's change this val type to integer and persist it with encode/decode. Just in case we upgrade the format once again in the future.